### ISSUE TITLE
Form control with "slider" role has no associated label

### ISSUE THREAD

**Issue by sunilt4** [IBM/carbon-components-angular#1846]
## Detailed description

> Describe in detail the issue you're having.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

dual-slider.component.html

> What did you expect to happen? What happened instead? What would you like to see changed?

No violation for Accessibility checker

> What browser are you working in?

Chrome (Version 91.0.4472.77)

> What version of the Carbon Design System are you using?

4.48.3

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
ADDI

## Additional information

* Screenshots or code
*![image](https://user-images.githubusercontent.com/44821670/121731329-c99c6900-cabe-11eb-9c59-2ed4a85e602f.png)
*![image](https://user-images.githubusercontent.com/44821670/121731414-e638a100-cabe-11eb-82f9-47f92295ff90.png)
*![image](https://user-images.githubusercontent.com/44821670/121731766-5f37f880-cabf-11eb-9b59-a979546b05b7.png)

* Notes
* Inside the labels I tried adding in the **for** attribute along with using data binding where I assigned it to the **id** variable. **[for]="id"**
* There are **2 labels**, **one for when the slider is low(zoomed out)**, and **one for when the slider is high(zoomed in)**.
* However, I tried manually adding in **for="slider-0"** in the labels within the inspect html element in chrome dev tools and the violations went away (Please refer to the last screenshot).

**Comment by sunilt4** [comment]
Hello team, I was instructed if we can target this issue to be completed during this release 6.0.1.

**Comment by cal-smith** [comment]
> Hello team, I was instructed if we can target this issue to be completed during this release 6.0.1.

We'll try to take a look. I'm not sure what 6.0.1 means though, I have no idea what product you work on. If you have some target dates that would be great.

**Comment by sunilt4** [comment]
Hello Cal. My team and I are working on the product ADDI (Application Discovery and Delivery Intelligence). I am working on ADDI-X which is the extensions part of this product. 6.0.1 is our next version release for this product. We have ADDI 6.0.1 defect fixing deadline by 7/02/2021.

**Comment by stanislavgeorgiev** [comment]
@cal-smith I'm starting to look at this issue now

**Comment by cal-smith** [comment]
> @cal-smith I'm starting to look at this issue now

Thanks @stanislavgeorgiev!
